changelog/v1.7.0-beta17/regex-rewrite.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/3321
+    description: Add regex rewrite option to routes
```

Since these changelogs are user facing (they become [part of our docs](https://docs.solo.io/gloo-edge/latest/reference/changelog/open_source/)), we should try to make them as descriptive as possible (within reason). Here I would add at least the new field name, e.g.
```suggestion
    description: Add new `regexRewrite` option to routes. This new field can be used to [...]
```